main.go

```diff
@@ -86,28 +89,47 @@ func main() {
 
 	restConfig := ctrl.GetConfigOrDie()
 
-	apiExportName := os.Args[1]
-	setupLog = setupLog.WithValues("api-export", apiExportName)
-
-	setupLog.Info("Looking up virtual workspace URL")
-	cfg, err := restConfigForAPIExport(ctx, restConfig, os.Args[1])
-	if err != nil {
-		setupLog.Error(err, "error looking up virtual workspace URL")
-	}
-
-	setupLog.Info("Using virtual workspace URL", "url", cfg.Host)
-
-	mgr, err := kcp.NewClusterAwareManager(cfg, ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
-		HealthProbeBindAddress: probeAddr,
-		LeaderElection:         enableLeaderElection,
-		LeaderElectionID:       "68a0532d.my.domain",
-	})
-	if err != nil {
-		setupLog.Error(err, "unable to start manager")
-		os.Exit(1)
+	setupLog = setupLog.WithValues("api-export-name", apiExportName)
+
+	var mgr ctrl.Manager
+	var err error
+	if apisKcpGroupPresent(restConfig) {
+
+		setupLog.Info("Looking up virtual workspace URL")
+		cfg, err := restConfigForAPIExport(ctx, restConfig, apiExportName)
+		if err != nil {
+			setupLog.Error(err, "error looking up virtual workspace URL")
+		}
+
+		setupLog.Info("Using virtual workspace URL", "url", cfg.Host)
+
+		mgr, err = kcp.NewClusterAwareManager(cfg, ctrl.Options{
```

Comment:
Pull ctrl.Options out to a variable so it's not duplicated?

Reply:
I made that change, but somehow forgot to push it :man_facepalming: 
changed in a93e0ba